ew/static/items.py

```diff
@@ -3535,6 +3573,28 @@
         furniture_place_desc = "You chuck the pillow on your bedding of choice. You're going to sleep *damn* well tonight.",
         furniture_look_desc = "A pillow stuffed with red glowing feathers magnificently rests upon their bed.",
     ),
+    EwFurniture(
+        id_furniture = "honeycombcouch",
+        str_name = "Honeycomb Couch",
+        str_desc = "A hexagonal couch made entirely from honeycombs. It's sticky, and covered bees, and, uh... not couch-shaped.",
+        rarity = "Patrician",
+        acquisition = "bartering",
+        price=45,
+        vendors=[ewcfg.vendor_atomicforestpp],
+        furniture_place_desc = "You set the honeycombs on the floor, crushing a few bees in the process. Honey slowly starts dripping off of it.",
+        furniture_look_desc = "You pick up the honeycombs from the floor, angering a few bees. Better hope you're not allergic to them!",
```

Comment:
The furniture_look_desc variable is used when you !look in your apartment not when you pick up a piece of furniture. this would be a pretty long string for that purpose